queries.sql

```diff
@@ -160,4 +159,97 @@ where
     date_of_birth BETWEEN '1990-01-01'
     AND '2000-12-31'
 group by
-    species;
\ No newline at end of file
+    species;
+
+/* What animals belong to Melody Pond? */
+select
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+from
+    animals
+    join owners on animals.owner_id = owners.id
+where
+    owners.full_name = 'Melody Pond';
+
+/* List of all animals that are pokemon (their type is Pokemon) */
+select
+    animals.name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+from
+    animals
+    join species on animals.species_id = species.id
+where
+    species.name = 'Pokemon';
+
+/* List all owners and their animals, remember to include those that don't own any animal */
+select
+    owners.full_name,
+    animals.name
+from
+    animals
+    right join owners on animals.owner_id = owners.id;
+
+/* How many animals are there per species? */
+/* Pokemon */
+select
+    species.name,
+    animals.name
+from
+    animals
+    join species on animals.species_id = species.id
+where
+    species.name = 'Pokemon';
+
+/* Digimon */
+select
+    species.name,
+    animals.name
+from
+    animals
+    join species on animals.species_id = species.id
+where
+    species.name = 'Digimon';
```

Comment:
👍 Good job on your queries! However, there's a more efficient way to achieve the same result as per the project requirement:

- [ ] As per the project requirement to answer "How many animals are there per species?", you can use a single query that combines the power of COUNT and GROUP BY. This query will provide a more efficient and organized way to answer the question, especially when dealing with a larger dataset. Here's the suggested query:

```sql
SELECT species.name, COUNT(animals.id) AS animal_count
FROM animals
JOIN species ON animals.species_id = species.id
GROUP BY species.name;
```

This query calculates the number of animals for each species and groups the results by species name. It's a more streamlined way to obtain the same result. Consider updating your code with this query to meet the project requirements. Keep up the great work! 👍📋📊